cli/pin.ml

```diff
@@ -1,8 +1,12 @@
 open Duniverse_lib
 open Rresult
 
+let confirm_replace_pin name () =
+  Prompt.confirm_or_abort ~yes:false ~question:(fun l ->
```

Comment:
I think it might be nice to add the `yes` flag to the command and to pass it here as well.